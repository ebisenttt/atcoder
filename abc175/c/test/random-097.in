-477395982176131 988344449940688 255444981950874
