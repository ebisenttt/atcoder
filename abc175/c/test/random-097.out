33493981725617
